enterprise/server/test/integration/workflow/workflow_test.go

```diff
@@ -86,6 +122,26 @@ func setup(t *testing.T, gp interfaces.GitProvider) (*rbetest.Env, interfaces.Wo
 	return env, workflowService
 }
 
+func triggerWebhook(t *testing.T, gitProvider *testgit.FakeProvider, workflowService interfaces.WorkflowService, repoURL string, commitSHA string, webhookURL string) {
+	// Set up the fake git provider so that GetFileContents can return the
+	// buildbuddy.yaml from our test repo
+	gitProvider.FileContents = repoContents()
```

Comment:
nit: it would be less error prone to configure this in each test -- since for some tests you want the "slow" repo contents to be returned here

Reply:
Good catch. Modified the helper function to take file contents as input. Thought it would be nice to have it in a helper function so we don't have to duplicate the descriptive comments in every test